search-app/docs/api.md

```diff
@@ -1973,19 +1971,32 @@ By concept id and revision id
 
     curl -i "%CMR-ENDPOINT%/concepts/:concept-id/:revision-id"
 
-Examples:
+Plain examples, with and without revision ids:
 
     curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1"
-    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1.iso"
-    curl -i -H 'Accept: application/xml' "%CMR-ENDPOINT%/concepts/G100000-PROV1"
-    curl -i -H 'Accept: application/metadata+xml' "%CMR-ENDPOINT%/concepts/G100000-PROV1"
-    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1.json"
     curl -i "%CMR-ENDPOINT%/concepts/C100000-PROV1/1"
-    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1/2.echo10"
     curl -i "%CMR-ENDPOINT%/concepts/V100000-PROV1"
     curl -i "%CMR-ENDPOINT%/concepts/V100000-PROV1/1"
     curl -i "%CMR-ENDPOINT%/concepts/V100000-PROV1/2"
 
+File extension examples:
+
+    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1.iso"
+    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1.json"
+    curl -i "%CMR-ENDPOINT%/concepts/G100000-PROV1/2.echo10"
+    curl -i "%CMR-ENDPOINT%/concepts/V100000-PROV1.umm_json"
+    curl -i "%CMR-ENDPOINT%/concepts/V100000-PROV1/2.umm_json"
+    curl -i "%CMR-ENDPOINT%/concepts/V100000-PROV1/2.umm_json_v1_9"
+
+MIME-type examples:
+
+    curl -i -H 'Accept: application/xml' \
```

Comment:
Nitpick: Copying multiline terminal commands is typically pretty wonky for me. I think this is one case where longer lines are totally okay - in fact, I prefer them for that^ reason. Granted this is just an example command, so feel free to ignore me